packages/eslint-plugin-template/src/utils/get-loc-from-absolute-source-span.ts

```diff
@@ -0,0 +1,12 @@
+import type { AbsoluteSourceSpan } from '@angular/compiler';
+import type { TSESLint, TSESTree } from '@typescript-eslint/experimental-utils';
+
+export function getLocFromAbsoluteSourceSpan(
```

Comment:
Not sure if it's the best name + the best place, but I do think it's a good ideia to have this utility (we can refactor the rules to use it in another PR).